packages/eas-cli/src/graphql/queries/BuildQuery.ts

```diff
@@ -0,0 +1,63 @@
+import gql from 'graphql-tag';
+
+import { QueryBuilder, createFragment } from '../queryTools';
+import { Build } from '../types/Build';
+
+type Filters = Partial<Pick<Build, 'platform' | 'status'>> & {
+  order?: number;
+  limit?: number;
+};
+
+type ArtifactFragmentType = Pick<Build, 'artifacts'>;
+
+export const BuildFragments = {
+  Artifacts: createFragment<Build, ArtifactFragmentType>(
+    'ArtifactsFragment',
+    'fragment ArtifactFragment on Build { artifacts { buildUr } }'
+  ),
+};
+
+export class BuildQuery {
+  static byId(buildId: string) {
```

Comment:
```suggestion
export namespace BuildQuery {
  export function byId(buildId: string): QueryBuilder<Build> {
```
I don't have a strong opinion on that, but if you just want force users  of this API to use certain names  and you are never instantiating this class namespaces are more appropriate

cc @dsokal

Reply:
I like the idea, however we don't use namespaces anywhere else and I'm not sure if we should.

Reply:
Let's stay with the class + static functions approach.